src/hotspot/share/opto/chaitin.hpp

```diff
@@ -143,7 +143,7 @@ class LRG : public ResourceObj {
 
 private:
   // Number of registers this live range uses when it colors
-  uint16_t _num_regs;           // 2 for Longs and Doubles, 1 for all else
+  uint16_t _num_regs;           // byte size of the value divided by slot size which is 4
```

Comment:
Is this true for oops, too?  Hadn't they been mapped to one slot on both, 32 and 64-bit platforms?